src/components/navBar/navlink.js

```diff
@@ -0,0 +1,38 @@
+import styles from '@/components/navBar/navBar.module.scss';
+
+const links = [
+    {
+      id:1,
+      name:'Home',
+      link:'https://www.realdevsquad.com/',
+      tabStyle:styles.homeTab
+    },
```

Comment:
NIT: Add spacing between key and value
```suggestion
    {
      id: 1,
      name: 'Home',
      link: 'https://www.realdevsquad.com/',
      tabStyle :styles.homeTab
    },
```